sample-app/README.md

```diff
@@ -0,0 +1,227 @@
+# Sample Application
+
+This is a sample application that demonstrates various aspects of interacting
+with HashiCorp [Vault][vault], including:
+
+- [AppRole][vault-app-role] authentication with a [response-wrapping token][vault-token-wrapping]
+- Reading a static secret from [kv-v2 secrets engine][vault-kv-v2]
+- Reading a dynamic secret from [PostgreSQL database secrets engine][vault-postgresql]
+- Periodic token [lease renewal][vault-leases] & re-login logic
+- Periodic database credentials [lease renewal][vault-leases] & reconnection logic
```

Comment:
Should we use a different adjective since `periodic` might be confused with [the 2nd example here](https://www.vaultproject.io/docs/commands/token/create#examples)

Reply:
Great catch! I'll fix this